backend/eks/deploy.sh

```diff
@@ -14,22 +14,28 @@ if ! aws sts get-caller-identity ; then
     aws configure
 fi
 
-git clone https://github.com/SUSE/cap-terraform.git -b cap-ci
+git clone https://github.com/SUSE/cap-terraform.git -b bisingh/eks
```

Comment:
Could we merge bisingh/eks into cap-ci and consume that?

Reply:
changes merged.